petastorm/tests/test_pytorch_dataloader.py

```diff
@@ -9,10 +8,12 @@
 import torch
 
 from petastorm import make_reader, TransformSpec, make_batch_reader
-from petastorm.pytorch import _sanitize_pytorch_types, DataLoader, BatchedDataLoader, decimal_friendly_collate
+from petastorm.pytorch import (_sanitize_pytorch_types, DataLoader,
+                               BatchedDataLoader, decimal_friendly_collate,
+                               InMemBatchedDataLoader, _load_rows_into_mem)
 from petastorm.tests.test_common import TestSchema
 
-ALL_DATA_LOADERS = [DataLoader, BatchedDataLoader]
+OOM_DATA_LOADERS = [DataLoader, BatchedDataLoader]
```

Comment:
Does OOM stands for out-of-memory? Not sure what would be a better name, but it's a bit strange to see it here :)

Reply:
Or name them as `BASIC_DATA_LOADERS`?

Reply:
That works. Just was surprised to see an OOM here :)